nuxt.config.ts

```diff
@@ -38,6 +38,7 @@ export default defineNuxtConfig({
   routeRules: {
     '/bylaws': { redirect: 'https://docs.google.com/document/d/1i6oXabir-628csa2bQacHpc17Kmhec9EfkyHij9yQGM/edit' },
     '/2021-survey': { redirect: 'https://docs.google.com/forms/d/e/1FAIpQLSeUyguShgoj6TZmqTboJN8qFVjKagrn1WRFvOuqDHqyGHEWeA/viewform?usp=sf_link' },
+    '/api/events': { cors: true },
```

Comment:
This is what I was going to commit as well. So looks good to me and we will not have any issues with this even if it doesn't work.

I was doing a bit of reading where some folks said they had to add some middleware for the cors to work properly.

+1 LGTM

Reply:
NVM it looks like you tested it and it works https://github.com/devedmonton/DES-Website/pull/346#issuecomment-2127760910